lib/workload/stateful/event_source/component.ts

```diff
@@ -55,7 +55,10 @@ export class EventSource extends Construct {
   constructor(scope: Construct, id: string, props: EventSourceProps) {
     super(scope, id);
 
-    this.deadLetterQueue = new Queue(this, 'DeadLetterQueue', { enforceSSL: true });
+    this.deadLetterQueue = new Queue(this, 'DeadLetterQueue', {
+      queueName: 'orcabus-event-source-dlq',
```

Comment:
Hardcoded queue name should probably go under constants/props?

Reply:
I thought `constants.ts` was getting a bit long already. I hard-coded it because there is no requirement in the current stack to set this as configurable, i.e. there is no dependency from the stateful stack to the stateless stack for the name of the dead letter queue.

Although probably for symmetry with the non-dead letter queue this could be configurable?

Reply:
Perhaps, suffix `-dlq` to the corresponding main queue name. i.e. `props.queueName + '-dlq'` something like that...

Reply:
Yes, I think that's a good in between.